src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/Manage Statistics/ManageStatisticsCardView.swift

```diff
@@ -0,0 +1,167 @@
+		layer.borderColor = borderColor.cgColor
+		accessibilityIdentifier = AccessibilityIdentifiers.LocalStatistics.manageStatisticsCard
+	}
+	
+	// MARK: - Internal
+
+	// swiftlint:disable:next function_parameter_count
+	func configure(
+		localStatisticsModel: LocalStatisticsModel,
+		availableCardsState: LocalStatisticsState,
+		onAddStateButtonTap: @escaping (SelectValueTableViewController) -> Void,
+		onAddDistrict: @escaping (SelectValueTableViewController) -> Void,
+		onDismissState: @escaping () -> Void,
+		onDismissDistrict: @escaping (Bool) -> Void,
+		onFetchGroupData: @escaping (LocalStatisticsDistrict) -> Void,
+		onEditButtonTap: @escaping () -> Void,
+		onAccessibilityFocus: @escaping () -> Void
+	) {
+		self.model = localStatisticsModel
+
+		self.onAddStateButtonTap = onAddStateButtonTap
+		self.onAddDistrict = onAddDistrict
+		self.onDismissState = onDismissState
+		self.onDismissDistrict = onDismissDistrict
+		self.onFetchGroupData = onFetchGroupData
+		self.onEditButtonTap = onEditButtonTap
+		self.onAccessibilityFocus = onAccessibilityFocus
+
+		updateUI(for: availableCardsState)
+	}
+
+	@IBAction func onAddLocalIncidenceButtonPressed(_ sender: Any) {
+		guard let model = self.model else {
+			Log.warning("AddStatistics model is nil", log: .localStatistics)
+			return
+		}
+		viewModel = ManageStatisticsCardsViewModel(
+			localStatisticsModel: model,
+			presentFederalStatesList: { selectedStateValueViewModel in
+				self.presentAddLocalStatistics(selectValueViewModel: selectedStateValueViewModel)
+			},
+			presentSelectDistrictsList: { selectedDistrictValueViewModel in
+				self.presentAddLocalStatisticsDistrict(selectValueViewModel: selectedDistrictValueViewModel)
+				
+			}, onFetchGroupData: { [weak self] district in
+				self?.onFetchGroupData?(district)
+			}
+		)
+		viewModel?.presentStateSelection()
+	}
+
+	func updateUI(for state: LocalStatisticsState) {
+		// clear
+		stackView.arrangedSubviews.forEach { subview in
+			stackView.removeArrangedSubview(subview)
+			subview.removeFromSuperview()
+		}
+
+		let addView = { () -> CustomDashedView in
+			let add = CustomDashedView.instance(for: .add)
+			add.tapHandler = { [weak self] in
+				Log.debug("add…", log: .ui)
+				self?.onAddLocalIncidenceButtonPressed(add)
+			}
+			add.label.onAccessibilityFocus = onAccessibilityFocus
+			return add
+		}()
+
+		let modifyView = { () -> CustomDashedView in
+			let modify = CustomDashedView.instance(for: .modify)
+			modify.tapHandler = { [weak self] in
+				Log.debug("modify…", log: .ui)
+				self?.onEditButtonTap?()
+			}
+			modify.label.onAccessibilityFocus = onAccessibilityFocus
+			return modify
+		}()
+
+		switch state {
+		case .empty:
+			// just 'add'
+			stackView.addArrangedSubview(addView)
+		case .notYetFull:
+			// 'add' & 'modify'
+			stackView.addArrangedSubview(addView)
+			stackView.addArrangedSubview(modifyView)
+		case .full:
+			// just 'modify'
+			stackView.addArrangedSubview(modifyView)
```

Comment:
Shouldn't the "Lokale 7-Tage-Inzidenz hinzufügen" Button be greyed out instead of removed? Looks like that in Figma and Android also had it that way this morning.